server/src/leap/soledad/server/auth.py

```diff
@@ -411,6 +412,9 @@ def _verify_token_in_couch(self, uuid, token):
                                       either missing in the tokens db or is
                                       invalid.
         """
+        cache = get_cache_for('authorized', expire=120)
+        if uuid in cache:
```

Comment:
Awesome that you found that! \o/
I am more annoyed that this doesn't broke any test.
Will add one before fixing to make sure that this kind of change doesn't pass on future.